### ISSUE TITLE
Quick start guide should link to prerequisites ~1

### ISSUE THREAD

**Issue by phillipskevin** [donejs/donejs#545]
If you jump right to the Quick Start guide, you won't realize there steps that should be done first.

It would be nice to have a link from the "Setup" section of the Quick Start Guide to this page: https://donejs.com/SettingUp.html

**Comment by matthewp** [comment]
This was done in b51f58d4098d6fed68f868c78c69e8e0e9bf652b
